example/agents_example.dart

```diff
@@ -15,7 +15,7 @@ void main() async {
     agentId: agentId,
     messages: [
       UserMessage(
-        content: UserMessageContent.string('What is the best French cheese?'),
+        content: 'What is the best French cheese?',
```

Comment:
In [ba6b353](https://github.com/nomtek/mistralai_client_dart/pull/102/commits/ba6b3530104542ae8c91935fae1cde8198e53cd1) I changed the UserMessage implementation so that it does not introduce breaking changes any more.